README.md

```diff
@@ -134,7 +134,9 @@ When creating tables you can pass specific throughput settings or stream specifi
 
 ```js
 dynogels.createTables({
-  'BlogPost': {readCapacity: 5, writeCapacity: 10},
+  'BlogPost': {
+    billingMode: 'PAY_PER_REQUEST'
```

Comment:
Since this part of documentation is not specifically about 'on-demand' mode, maybe leave as it was before?

Reply:
Let’s add one example for each option so the developer can see what is possible

PAY_PER_REQUEST is super convenient